src/catalog/src/builtin.rs

```diff
@@ -2601,6 +2601,15 @@ pub static MZ_INTERNAL_CLUSTER_REPLICAS: Lazy<BuiltinTable> = Lazy::new(|| Built
     access: vec![PUBLIC_SELECT],
 });
 
+pub static MZ_UNSTABLE_CLUSTER_REPLICAS: Lazy<BuiltinTable> = Lazy::new(|| BuiltinTable {
+    name: "mz_unstable_cluster_replicas",
```

Comment:
I don't understand what this name is supposed to tell me.

Reply:
https://github.com/MaterializeInc/materialize/pull/27904#discussion_r1667228503